hazelcast/src/test/java/com/hazelcast/test/starter/test/HazelcastVersionLocatorTest.java

```diff
@@ -51,6 +52,8 @@ public class HazelcastVersionLocatorTest {
 
     @Test
     public void testDownloadVersion() throws Exception {
+        // TODO This test doesn't force a redownload, so if an artifact is cached in the local repository, the download won't be
+        // excercised. It's difficult to modify the local maven repository as it's not encapsulated for the scope of testing
         Map<HazelcastVersionLocator.Artifact, File> files = HazelcastVersionLocator.locateVersion("4.0", true);
```

Comment:
> Do we want to update this for 5.0? Makes sense imo

I don't see any value in changing it to v5? This test is testing a test utility, that it can download an arbitrary version of the required dependency successfully.

When it's used it's typically testing that older (i.e. v3 or v4) instances can be communicated with so it's fine I think.

Reply:
Just makes sense to keep it updated with our latest major version, since older major version will be forgotten about eventually - not a big deal at all though, happy to approve without as like you said the focus is on downloading _an artefact_ more than specifics.